Tests/LibWeb/Text/expected/css/getComputedStyle-print-all.txt

```diff
@@ -86,6 +86,8 @@ clip-rule: nonzero
 color: rgb(0, 0, 0)
 column-count: auto
 column-gap: auto
+column-span: all
+column-width: auto
```

Comment:
It's a pain I know, but please update this test result in each commit that adds a property, so that it always passes. Sometimes it's necessary to go back to old commits to test when something broke, and it's a lot less confusing if unrelated tests don't break.